### DISCUSSION TITLE
Web Service Distribuição CTe

### DISCUSSION THREAD

**Discussion by denisrdasilva** [wmixvideo/nfe#753] [Geral] [answered]
Olá pessoal! Ao consumir o Web Service Distribuição CTe, é retornado tanto os CTe's como os eventos vinculados. Não consegui achar a classe que representa os eventos, de forma genérica. Se realmente não existir, posso cria-la aqui e depois comitar? Assim até iniciaria a contribuir para o projeto. 

![evento_imagem](https://user-images.githubusercontent.com/36534256/124325793-ad3e9a00-db5b-11eb-8a1e-88312b03acbe.PNG)

**Comment by fincatto** [comment, marked as answer]
Sim. Se houver um exemplo funcional, vamos adicionar ao projeto!
É importante que esteja funcional e que siga as guidelines do projeto, sempre acompanhado dos testes unitarios.
